src/Microsoft.TestPlatform.PlatformAbstractions/net451/System/PlatformEnvironment.cs

```diff
@@ -61,5 +61,18 @@ public int GetCurrentManagedThreadId()
         {
             return Thread.CurrentThread.ManagedThreadId;
         }
+
+        /// <inheritdoc />
+        public bool GetEnviromentVariable(string envVar, ref double value)
+        {
+            var isValueSet = false;
+            var envVarValue = Environment.GetEnvironmentVariable(envVar);
+            if (!string.IsNullOrEmpty(envVarValue) && double.TryParse(envVarValue, out value))
+            {
+                isValueSet = true;
+            }
+
+            return isValueSet;
+        }
```

Comment:
Not required in Abstraction. 
For testing, either have EnvironmentHelper or directory use it in the tests.

Reply:
Done.